src/cascadia/TerminalApp/pch.h

```diff
@@ -25,40 +25,39 @@
 
 #include <wil/cppwinrt.h>
 
-#include <unknwn.h>
-
-#include <hstring.h>

```

Comment:
Since I had to add `<winrt/Windows.Globalization.h>` to some pch.h files I decided to cleaned them up by removing unused includes and ordering and reordered the includes alphabetically.

Reply:
be careful! you must include unknwn and hstring _BEFORE_ any c++/winrt headers; `wil` is _also_ sensitive to include order (!)

Reply:
I removed those entirely, because it compiles without them.